detectors/node/opentelemetry-resource-detector-aws/src/detectors/AwsEcsDetector.ts

```diff
@@ -62,9 +62,9 @@ export class AwsEcsDetector implements Detector {
     const metadataResource =
       containerAndHostnameResource.merge(metadatav4Resource);
 
-    if (!metadataResource.attributes) {
-      return Resource.empty();
-    }
+    // if (!metadataResource.attributes) {
+    //   return Resource.empty();
+    // }
```

Comment:
Are you sure this is meant to be left like that?

Reply:
No it’s not, good catch :-)

Reply:
Done in 2f79d48a9ecf97948a158065ac9ed3619cee9a5e